website/docs/language/settings/terraform-cloud.html.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: "language"
+page_title: "Terraform Settings - Configuration Language"
+sidebar_current: "docs-config-terraform"
+description: "The terraform block allows you to configure Terraform behavior, including the Terraform version, backend, integration with Terraform Cloud, and required providers."
+---
+
+# Terraform Cloud
```

Comment:
```suggestion
# Terraform Cloud Configuration
```

Mostly just: a page whose title is simply "Terraform Cloud" is a bit disorienting. Better to risk being over-verbose to make sure people know where they are.